crates/re_log_types/src/path/entity_path.rs

```diff
@@ -154,6 +154,12 @@ impl EntityPath {
         self.path.is_root()
     }
 
+    /// Is this equals to, or a descendant of, the given path.
+    #[inline]
+    pub fn starts_with(&self, prefix: &EntityPath) -> bool {
```

Comment:
Nit: just based on name ambiguity as to whether this allows for partial match on the tail element.

Something like: `is_self_or_descendant_of` is more clear but a bit too verbose :thinking:

Reply:
`starts_with` matches the name in the standard library for paths, slices, and strings:

* https://doc.rust-lang.org/stable/std/primitive.slice.html#method.starts_with
* https://doc.rust-lang.org/stable/std/primitive.str.html#method.starts_with
* https://doc.rust-lang.org/stable/std/path/struct.Path.html#method.starts_with